### DISCUSSION TITLE
Problemi con servizio camera.snap

### DISCUSSION THREAD

**Discussion by bondeste** [indomus/forum#1708] [HUB: Home Assistant]
Ciao a tutti, sto cercando di salvare un immagine dalla mia ipcam utilizzando il servizio "camera.snapshot".
Ma purtroppo non riesco a capire come mai non mi crea nessun tipo di file.
Sto utilizzando lo strumento sviluppatori per chiamare il servizio, di seguito il codice che sto utilizzando:
`service: camera.snapshot
target:
  entity_id: camera.citofono
data:
  filename: '/media/local/prova.jpg'`
utilizzo HA su docker.
Grazie mille in anticipo per l'aiuto.

**Comment by Scialla** [comment]
Hai mappato quel percorso quando crei il container di HA? Altrimenti se lo crea internamente nel container

**Comment by bondeste** [comment]
Si, utilizzo quella cartella per andare a pescare delle immagini quando si scatena un determinato evento e tutto funziona.
